### PR TITLE
Renamed `bsz` to `bs` for consistency; removed dead code

### PR DESCRIPTION
**Pull request by awgu**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #299

some minor cleanups

### CODE REVIEW

File: torchtitan/models/llama/model.py [hunk 1]
```diff
@@ -132,7 +132,6 @@ class Attention(nn.Module):
     Attributes:
         n_kv_heads (int): Number of key and value heads.
         n_heads (int): Number of query heads.
-        n_local_kv_heads (int): Number of local key and value heads.
```
Comment (awgu) [inline torchtitan/models/llama/model.py:135]: not an attribute (only one occurrence of `n_local_kv_heads` if you search in this file)

File: torchtitan/models/llama/model.py [hunk 2]
```diff
@@ -183,12 +182,12 @@ def forward(
             torch.Tensor: Output tensor after attention.
 
         """
-        bsz, seqlen, _ = x.shape
```
Comment (awgu) [inline torchtitan/models/llama/model.py:186]: all inline comments in this method use `bs` for batch size so can make this `bs` for consistency

File: torchtitan/models/llama/model.py [hunk 3]
```diff
@@ -421,7 +420,7 @@ def forward(self, tokens: torch.Tensor):
             torch.Tensor: Output logits after applying the Transformer model.
 
         """
-        _bsz, seqlen = tokens.shape
```
Comment (awgu) [inline torchtitan/models/llama/model.py:424]: similarly, `_bsz` is unused, so just remove

Comment (awgu) [reply torchtitan/models/llama/model.py:424]: if it helps readability to know the `tokens.shape` is (batch size, sequence length), I can keep it and maybe rename it to `_bs`?

Comment (tianyu-l) [reply torchtitan/models/llama/model.py:424]: Although not used, it improves code readability -- it tells how many dimensions `tokens` has, and what they are. So IMO I'd wish they are kept. Also, the "unusedness" has been indicated using the `_` prefix.

**Review by tianyu-l — APPROVED** [review]
One comment inline.

File: torchtitan/models/llama/model.py (hunk 3 above)
Comment (tianyu-l) [reply torchtitan/models/llama/model.py:424]: > if it helps readability to know the `tokens.shape` is (batch size, sequence length), I can keep it and maybe rename it to `_bs`?

just saw this message, yeah I agree

Comment (awgu) [reply torchtitan/models/llama/model.py:424]: changed it to `_bs`

**Review by wanchaol — APPROVED** [review]

**Pull request merged**